src/dbnode/persist/fs/commitlog/README.md

```diff
@@ -0,0 +1,73 @@
+# Overview
+
+The commitlog package contains all the code for reading and writing commitlog files.
+
+# Implementation
+
+## Writer
+
+The commitlog writer consists of two main components:
+
+1. The `commitlog` (`commit_log.go`) itself which contains all of the public APIs as well as the logic for managing concurrency, rotating files, and inspecting the status of the commitlog (via the `ActiveLogs` and `QueueLength` APIs).
```

Comment:
nit: Remove the file names since they are much more likely to change.

The types you could use is `CommitLog` for the commit log and `CommitLogWriter` (even though the latter is not exported).

Both these symbols are way more grep-able than the two you're using at the moment (just my 2 cents).

Reply:
good feedback, I removed all file names and change all references to be `CommitLog` or `CommitLogWriter`